docs/components/DosAndDonts/DosAndDonts.tsx

```diff
@@ -30,20 +30,16 @@ export const DoOrDont = ({
   <div className={classnames("relative", className)} {...otherProps}>
     <div
       className={classnames(
-        "absolute top-6 left-6 z-10 py-4 px-6 font-bold text-white rounded flex gap-2 items-center justify-center",
+        "absolute top-6 left-6 z-10 py-2 ps-4 pe-8 font-bold text-white rounded flex gap-2 items-center justify-center",
         isDont ? "bg-red-500" : "bg-green-500"
       )}
     >
-      {isDont ? (
-        <CloseIcon role="presentation" />
-      ) : (
-        <CheckIcon role="presentation" />
-      )}
-      <SbContent>
-        <span className="text-white">
+      <Unstyled style={{ display: "contents" }}>
```

Comment:
We actually use it here as intended - Unstyled stops SB global styles leaking through, but I don't want the actual div to exist because it affects the layout.
I'll experiment later to see if I can move the Unstyled elsewhere to not need to do this.